tests/tracer/test_settings.py

```diff
@@ -173,3 +177,29 @@ def test_x_datadog_tags(env, expected):
     with override_env(env):
         _ = Config()
         assert expected == (_._x_datadog_tags_max_length, _._x_datadog_tags_enabled)
+
+
+def test_settings_deprecation():
```

Comment:
```suggestion
@pytest.mark.subprocess(err=b"expected deprecation warning")
def test_settings_deprecation():
```

We should make this a subprocess test. This will ensure this test case (and the deprecated imports) are not impacted by other tests (we configure rate limits for ddtrace logs this helps us avoid generating multiple duplicate debug/warning/error logs in an application). Running this test in a subprocess will make this test more reliable and ignore potential conflicts with other test cases.